plugins/search/src/components/SearchModal/SearchModal.tsx

```diff
@@ -170,6 +170,17 @@ export const Modal = ({ toggleModal }: SearchModalProps) => {
   );
 };
 
+const Context: FC = ({ children }) => {
```

Comment:
avoid FC and SFC :) https://github.com/backstage/backstage/blob/master/docs/architecture-decisions/adr006-avoid-react-fc.md

Reply:
possible ideal: `function Context(props: { children?: React.ReactNode }) {...}` (avoids early unpacking, adds named function)

Reply:
Fixed, thanks 🙌🏻